### PR TITLE
chore: remove typescript files from .pre-commit-hooks.yaml

### PR DESCRIPTION
**Pull request by sid-kap**
Since typescript syntax is not supported by the linter, by default IMO we should not lint these files.

**What is the purpose of this pull request? (put an "X" next to item)**

[ ] Documentation update
[X] Bug fix
[ ] New feature
[ ] Other, please explain:

**What changes did you make? (Give an overview)**

Remove `.ts`/`.tsx` files from the pre-commit lint hook

**Which issue (if any) does this pull request address?**

In my mixed JavaScript/TypeScript project that uses pre-commit, I'm getting a syntax error (`Parsing error: Unexpected token :`) when `standard` tries to lint a TypeScript file

**Is there anything you'd like reviewers to focus on?**

### CODE REVIEW

**Review by Divlo — APPROVED** [review]

**Comment by sid-kap** [comment]
@Divlo looks like the same tests are also broken on master. Would it be okay to just merge this without fixing the tests in that case?

**Comment by Divlo** [comment]
> @Divlo looks like the same tests are also broken on master. Would it be okay to just merge this without fixing the tests in that case?

There is an open issue about the tests failing (see: #1663).
I guess it would be okay to merge this even if the tests are actually failing, as it is not related yes. 

Keep in mind that, this will not solve your problem : `I'm getting a syntax error (Parsing error: Unexpected token :) when standard tries to lint a TypeScript file`, if you are actually using TypeScript, you should take a look at [ts-standard](https://github.com/standard/ts-standard), if you are only linting JavaScript files, you should edit your own `.pre-commit-hooks.yaml` file, this PR will not affect in any way your project.

@sid-kap

**Comment by sid-kap** [comment]
Thanks!
Ah I see now, I didn't realize that ts-standard lints both typescript and javascript files. Anyway, I guess this will be useful in case someone wants to use `standard` for js and a different linter for ts files.

**Pull request merged**
